src/main/java/application/rest/v1/StatusEndpoint.java

```diff
@@ -73,6 +73,13 @@ public Response computeStatus(@Pattern(regexp = NAME_PATTERN_ONE_OR_MORE) @PathP
             @PathParam("resource-kind") @Parameter(description = "The Kubernetes resource kind for the resource") String kind,
             @Pattern(regexp = API_VERSION_PATTERN_ZERO_OR_MORE) @DefaultValue("") @QueryParam("apiversion") @Parameter(description = "The apiVersion of the resource") String apiVersion,
             @Pattern(regexp = NAME_PATTERN_ZERO_OR_MORE) @DefaultValue("default") @QueryParam("namespace") @Parameter(description = "The namespace of the resource") String namespace) {
+        if (Logger.isEntryEnabled()) {
```

Comment:
General comment:  I think we need to revisit the configmap caching.  It appears only configmaps from the kappnav ns (so-called 'global' ns) are cached.  The KAM support increases the likelihood (on purpose) that config maps come from other namespaces.  So caching should work for action/status/detail config maps that come from anywhere.

Reply:
There are actually two configmap caches, one for "global' ns, another one is for all configmaps, which is for actions and status mapping configmaps.

Reply:
I now recall the use of the GLOBAL_NAMESPACE constant is so the code caches config maps when the API server is running in kAppNav's namespace and does not cache with running in the actdev namespace.  There is no issue here.  Consider this resolved.